web/src/compose_ui.ts

```diff
@@ -1083,6 +1104,12 @@ export function format_text(
             }
             break;
         }
+        case "auto-surround": {
+            const syntax_start = inserted_content;
```

Comment:
Unrelated to the feature added here, but it would be nice if in a prep commit you could make this `inserted_content` variable optional instead of assigning it a default empty string value, which is never used